packages/contracts/contracts/Passport.sol

```diff
@@ -43,6 +46,34 @@ contract Passport is ERC721, Ownable {
     emit Purchase(msg.sender, price, _id, tokenURI(_id));
   }
 
+  function transfer(
+      address from,
+      address to,
+      uint256 tokenId
+  ) external payable {
+    enableTransfer = true;
+    if (royalty > 0) {
+      uint256 ownerPayment = royalty * 39 / 40;
+      (bool success1, ) = _owner.call{value: ownerPayment}("");
+      require(success1, "Address: unable to send value, recipient may have reverted");
+
+      uint256 cabinPayment = royalty / 40;
+      (bool success2, ) = _cabindao.call{value: cabinPayment}("");
+      require(success2, "Address: unable to send value, recipient may have reverted");
+    }
+    _transfer(from, to, tokenId);
```

Comment:
why not just call `super._transfer(from, to, tokenId)` here directly?

Reply:
We need to override _transfer...otherwise people can call ERC721's transferFrom or it's safe variants to transfer without paying royalty.

Reply:
Disclaimer: This is my first time with smart contracts and solidity, so my understanding might be a bit hazy about a few things :)

Reply:
right I understand overriding `transferFrom`, but I think overriding `_transfer` is redundant bc it's only being used here and is otherwise private

Also, we may want to override the `safeTransferFrom` method, as that eventually calls `_transfer`, and it would fail with `enableTransfer` being false